icons/tokens/white/ARIS.svg

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="500.000000pt" height="500.000000pt" viewBox="0 0 500.000000 500.000000"
```

Comment:
the size of the icon is too big, could this cause bug?
@valentunn @ERussel

Reply:
Usually we are using 48x48

Reply:
Good point, let's clarify that